packages/@okta/vuepress-site/docs/guides/submit-app/faqs/index.md

```diff
@@ -2,138 +2,98 @@
 
-The SCIM User Profile allows for an array of emails. The only way to differentiate between emails is to use the `type` sub-attribute.
+Currently, the App Wizard does not support extra login fields. Create an app using the Plug-in (SWA) Template Application. In order to promote your Template App to the Okta Integration Network, please email a screenshot with the details of the configured app to <developers@okta.com> with your app name in the subject line.
 
-* When returning multi-valued attributes, service providers should canonicalize the value returned (for example, by returning a value for the sub-attribute "type", such as "home" or "work") when appropriate (for example, for email addresses and URLs).
-* Service providers may return element objects with the same "value" sub-attribute more than once with a different `type` sub-attribute (for example, the same email address may be used for work and home) but should not return the same (type, value) combination more than once per attribute, as this complicates processing by the client.
-* When defining schema for multi-valued attributes, it is considered a good practice to provide a `type` attribute that may be used for the purpose of canonicalization of values. In the schema definition for an attribute, the service provider MAY define the recommended canonical values (see [RFC 7643 Section 7](https://tools.ietf.org/html/rfc7643#section-7)).
+### SAML
 
-See [Section 2.4 of RFC 7643](https://tools.ietf.org/html/rfc7643#section-2.4) for more details.
+**Q: Does Okta support single logout or single sign-out (SAML protocol)?**
 
-**I only have one email/phone number/address in my user profile. Do I need to implement the array of emails/phone numbers/addresses?**
+Yes. For more information, see the Advanced Settings section of the [Using the App Integration Wizard: SAML App Wizard](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_App_Integration_Wizard).
 
-Yes, you must return these fields in an array, which is specified in the SCIM spec as a multi-valued attribute: [Section 2.4](https://tools.ietf.org/html/rfc7643#section-2.4).
+**Q: I'm an ISV setting up a SAML 2.0 app using the App Wizard and we have different domains for each customer. How do you manage these types of situations?**
 
-**Why doesn't Okta support DELETE /Users?**
+Currently, the App Wizard does not support custom domains, but public-facing integrations can still have this functionality. Create an app integration as you normally would for a single domain using the App Wizard. After you get to the step of submitting your app integration through the [OIN Manager](https://oinmanager.okta.com), you will be asked to fill out a form with all the relevant information.
```

Comment:
**When** you get to the step of submitting your app integration through the [OIN Manager](https://oinmanager.okta.com), you **are** asked to fill out a form with all the relevant information.

Reply:
Fixed. Also changed the "does not" to "doesn't" in the first sentence..